resources/js/components/Common/FormButton.vue

```diff
@@ -33,24 +34,9 @@ function handleSubmit() {
 </script>
 
 <template>
-  <form :class="props.class" @submit.prevent="handleSubmit">
-    <button
-      v-if="!disabled && !processing"
-      type="submit"
-      class="font-semibold transition-colors duration-200 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-primary-600"
-      :class="`${small ? 'rounded-md text-xs 2xs:text-sm py-2' : 'rounded-lg py-3'} ${text ? 'text-black hover:text-primary-800 px-0 py-0' : 'px-2 2xs:px-3 bg-primary text-white hover:bg-primary-600 shadow-sm'} ${props.class}`"
-    >
-      <slot />
-    </button>
-
-    <button
-      v-else
-      disabled
-      type="submit"
-      class="font-semibold text-gray-500 cursor-not-allowed"
-      :class="`${small ? 'rounded-md text-xs 2xs:text-sm py-2' : 'rounded-lg py-3'} ${text ? 'px-0 py-0' : 'border border-gray-500 font-semibold px-2 2xs:px-3 '} ${props.class}`"
-    >
+  <form :class="`${processing ? 'cursor-wait' : ''} ${props.class}`" @submit.prevent="handleSubmit">
```

Comment:
You forgot to remove `props.class` or this is different class ? You can simple write `class` without define `props` for form.

Reply:
It was supposed to be different, but we actually don't use it anywhere. I will remove this prop.